<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgClean</class>
 <widget class="QDialog" name="DlgClean">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clean Options</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>10</y>
     <width>221</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Tip</string>
   </property>
   <widget class="QCheckBox" name="checkBoxTipCoordSys">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Recompute Coordinate System</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxTipSpike">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Spike Removal and Hole Filling</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>221</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Plate - Knife - Bullet</string>
   </property>
   <widget class="QCheckBox" name="checkBoxPltCoordSys">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Recompute Coordinate System</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxPltDetrend">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Detrend</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Data Islands</string>
   </property>
   <widget class="QCheckBox" name="checkBoxDataIslands">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Threshold Masking</string>
   </property>
   <widget class="QCheckBox" name="checkBoxThreshRun">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Texture Value 0-255:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Quality Value 0-255:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditThreshQMap">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>QMap:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowseQMap">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxThreshMinQl">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxThreshMinTx">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgClean</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgClean</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
